src/pages/home/components/Landing/Landing.jsx

```diff
@@ -1,15 +1,21 @@
-import React, { Component } from 'react';
+import React from 'react';
+import Header from '../../../../shared/components/Header/components/PageHeader/PageHeader';
 import BeerListContainer from '../BeerList/BeerListContainer';
 import Search from '../Search/Search';
+import Routes from '../../../../shared/constants/paths/paths';
```

Comment:
Let's write imported constants in UPPERCASE, so they won't look like imported React components